src/components/refactor/dialogs/two-windings-transformer-creation/phase-tap-changer-pane/phase-tap-changer-pane-utils.js

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {
+    CURRENT_LIMITER_REGULATING_VALUE,
+    ENABLED,
+    EQUIPMENT,
+    FLOW_SET_POINT_REGULATING_VALUE,
+    HIGH_TAP_POSITION,
+    ID,
+    LOW_TAP_POSITION,
+    NAME,
+    NOMINAL_VOLTAGE,
+    PHASE_TAP_CHANGER,
+    REGULATING,
+    REGULATION_MODE,
+    STEPS,
+    STEPS_ALPHA,
+    STEPS_CONDUCTANCE,
+    STEPS_RATIO,
+    STEPS_REACTANCE,
+    STEPS_RESISTANCE,
+    STEPS_SUSCEPTANCE,
+    STEPS_TAP,
+    SUBSTATION_ID,
+    TAP_POSITION,
+    TARGET_DEADBAND,
+    TOPOLOGY_KIND,
+    TYPE,
+    VOLTAGE_LEVEL,
+} from 'components/refactor/utils/field-constants';
+import { REGULATION_MODES } from '../../../../network/constants';
+import {
+    areArrayElementsOrdered,
+    areArrayElementsUnique,
+} from '../../../utils/utils';
+import yup from '../../../utils/yup-config';
+import {
+    getRegulatingTerminalEmptyFormData,
+    getRegulatingTerminalFormData,
+} from '../../regulating-terminal/regulating-terminal-form-utils';
+
+const phaseTapChangerValidationSchema = (id) => ({
```

Comment:
it would be better to put the validation in a file named phase-tap-changer-pane-validation-schema.js